### DISCUSSION TITLE
Onde encontrar profissionais que integram IA com Maketing e Vendas?

### DISCUSSION THREAD

**Discussion by MisaacBeach** [frontendbr/forum#2500] [Discussão]
Primeiro dia no get hub galera . Queria  saber se alguém sabe onde encontrar profissionais que já estejam adiantado na integração de IA no marketing e vendas de uma empresa ? Só encontrei até agora agências com preços astronômicos

**Comment by felquis** [comment]
A melhor forma é você se tornar um programador capaz de criar essas integrações. Estou estudando bastante coisa do assunto, mas não estou virando um ML Engineer apenas arquitetando soluções que usam um pouco de LLM para algumas categorizações e decisões que humanos tomariam olhando para qualquer pedaço de texto. Tem funcionado muito bem. Mas ainda não coloquei em prática em algum sistema, por enquanto apenas PoCs no meu lado

**Comment by felquis** [comment]
Notei que tem algumas pessoas desenvolvendo bots, que criam e respondem threads usando LLMs, e nas respostas puxam sardinha para algum produto ou serviço, é uma prática realmente deprimente de marketing que está sendo marcada como spam com alta frequência.
